IndicoToolkit.Tests/AutoClass/AutoClassifierTests.cs

```diff
@@ -68,20 +70,22 @@ public async Task TestToCSV()
             {
                 List<DocRecord> records = csv.GetRecords<DocRecord>().ToList();
                 Assert.Equal(3, records.Count());
-                foreach(DocRecord rec in records)
+                foreach (DocRecord rec in records)
                 {
                     Assert.Contains(".pdf", rec.filename);
                     if (rec.filename.Contains("yy"))
                     {
                         Assert.Equal("a", rec.docClass);
                         Assert.Contains("YY", rec.docText);
                         Assert.Contains("ZZ", rec.docText);
-                        shouldBeTwoAclass ++;
-                    } else {
+                        shouldBeTwoAclass++;
+                    }
+                    else
+                    {
                         Assert.Equal("b", rec.docClass);
                         Assert.DoesNotContain("YY", rec.docText);
                         Assert.DoesNotContain("ZZ", rec.docText);
-                        shouldBeOneBclass ++;
+                        shouldBeOneBclass++;
```

Comment:
should we be asserting what the count of these are at the end? Assert.Equal(1, shouldBeOneBclass)?

Reply:
line 92 and 93 should be doing that, although I can move them inside line 69